units/stats_task_end.go

```diff
@@ -27,11 +27,10 @@ func init() {
 }
 
 type collectTaskEndDataJob struct {
-	TaskID    string `bson:"task_id" json:"task_id" yaml:"task_id"`
-	Execution int    `bson:"execution" json:"execution" yaml:"execution"`
-	HostID    string `bson:"host_id" json:"host_id" yaml:"host_id"`
```

Comment:
We have to keep the `HostID` field (and the `j.host` logic in `Run`) temporarily until the app servers roll over and finish running the task end data jobs to maintain temporary backward compatibility with the existing task stats jobs already in the remote queue. We can attach a TODO to remove and then actually remove HostID after this commit is deployed.